### ISSUE TITLE
Strange loss values while training

### ISSUE THREAD

**Issue by hadi-ghnd** [endernewton/tf-faster-rcnn#125]
I'm trying to perform faster rcnn on a custom dataset based on pascal voc format. I have set the number of **iterations** to **50000** and started the training on a gpu with 4GB memory. Now I have two problems:

1) the loss values don't seem to get close to a particular number and they change randomly without any order:

![screenshot from 2017-06-09 13-39-26](https://user-images.githubusercontent.com/24352869/26969137-ea6d6ba6-4d19-11e7-854e-b1bdfe055193.png)

2) I don't know how to see if the network is over-fitting and if 50000 iterations are enough, low or even too much. How can I make sure how many iterations are appropriate?

**Comment by endernewton** [comment]
the loss value seems fine, different images have different numbers. 

you should use a separate validation set to test the model. if the model does not perform better after some iterations, then you are done.
